### DISCUSSION TITLE
Feature of advanced google search paramethers

### DISCUSSION THREAD

**Discussion by RomanAleynikov** [hoothin/SearchJumper#130] [Ideas]
I have an idea to new feature. Maybe is tere a sense to add google advanced search paramethers panel for engines? I use now this extension for this goal [Google Search Extra Buttons](https://greasyfork.org/en/scripts/7543-google-search-extra-buttons)
I imagine it like this:
![image](https://github.com/hoothin/SearchJumper/assets/87813216/69868d48-e25c-4fde-8e3f-f56660c55019)
or like this
![image](https://github.com/hoothin/SearchJumper/assets/87813216/4e39b137-9b29-48a1-8207-138ea17fb08c)

Result is:
![image](https://github.com/hoothin/SearchJumper/assets/87813216/a33d98fd-1114-4f63-93d1-441e97d9185d)

**Comment by hoothin** [comment]
Thank you for the suggestion.
That `Google advanced` is implemented using the `%input` parameter. You're free to add more functionality, but the advanced search parameters are different for each engine, so adding a universal advanced search template is not feasible.

Here is the example usage for `%input`:
`%input{Please select your cup/rare/excellent/So big,A/B/C}`
`Please select your cup` is the title.
`rare/excellent/So big` are the titles of first, second, third drop-down boxes.
`A/B/C` are the values of the drop-down boxes.

You can insert `%input` at any position to use it.
For example, `https://xxx.com?abc=%input{Please select your cup/rare/excellent/So big,A/B/C}&xyz=1`, after you click it, you will get a popup and `https://xxx.com?abc=A&xyz=1` when you selected `rare`.

**Reply by RomanAleynikov** [reply]
Ok. Thank you for detail explanation and examples.
